npm/pkg/transport/controlplane.go

```diff
@@ -119,11 +119,14 @@ func (m *Manager) start() error {
 				}
 			}
 		case <-m.ctx.Done():
-			klog.Info("Stopping transport manager")
+			klog.Info("Context Done. Stopping transport manager")
 			return nil
 		case err := <-m.errCh:
 			klog.Errorf("Error in transport manager: %v", err)
 			return err
+		case <-stopCh:
```

Comment:
Not related to this line but, NewManager and Manager is confusing, can we have the right names for this struct ? 

may be ControllerEventsManager ?

Reply:
If you read it using the fully-qualified path i.e. `transport.Manager` it makes sense to keep it this way. It states that this is our manager that encapsulates the server and watchdog as well. My suggestion is to keep it this way.

Reply:
right but searching for it will be a nightmare for new joinees, probably transport.TransportManager will be better,

Reply:
renamed to GrpcManager